### ISSUE TITLE
Dataset takes up only 1000 images

### ISSUE THREAD

**Issue by NavneetSajwan** [Holmeyoung/crnn-pytorch#49]
![Screenshot from 2020-04-24 08-03-47](https://user-images.githubusercontent.com/47170735/80169117-3ece8580-8602-11ea-839b-c0b3daecca90.png)
I have 1740 images. I tried to make a dataset using them. But it only chooses 1000 of all the images.

**Comment by ghost** [comment]
No. it created 1740 samples. please read the codes in create_dataset.py.
it just display the process every 1000 samples.
```
if cnt % 1000 == 0:
        writeCache(env, cache)
        cache = {}
        print('Written %d / %d' % (cnt, nSamples))
        cnt += 1
```

**Comment by ghost** [comment]
@NavneetSajwan 
```
train_dataset = lmdbDataset('path_to_your_created_lmdb')
print(len(train_dataset))
```
you will see the number of images

**Comment by NavneetSajwan** [comment]
thanks.

**Comment by NavneetSajwan** [comment]
How many epochs does it take to train on this data? I have done 500 epochs and still, I have just 12% accuracy.

**Comment by ghost** [comment]
you data is far from enough.
I used  http://www.robots.ox.ac.uk/~vgg/data/text/
Number of training images: 7,224,600
Number of validation images: 802,733 
7 epoch can achieve more than 88% accuracy on validation set.
